app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -4,14 +4,21 @@ def self.create_job(options)
     super(name, options.with_indifferent_access)
   end
 
+  attr_writer :minimize_indirect
```

Comment:
@bzwei Do you anticipate outside callers setting `:minimize_indirect`? If no then it could be changed to `attr_reader`.

Reply:
I am not sure how are we going to tune this property. @mkanoor what is your idea when you proposed to make it configurable?